assets/sass/05-blocks/search/_style.scss

```diff
@@ -52,12 +52,26 @@
 	button.wp-block-search__button {
 		margin-left: 0;
 		background-color: transparent;
-		color: var(--button--color-text-hover);
 		line-height: 1;
+		padding: var(--button--padding-vertical) calc(0.5 * var(--button--padding-horizontal));
+		color: inherit;
+		border: var(--button--border-width) solid currentColor;
 
+		.has-background &,
+		.has-background-dim & {
+			color: inherit;
+		}
+
+		&:active,
 		&:hover {
-			background-color: var(--button--color-background);
-			color: var(--button--color-text);
+			color: var(--button--color-text) !important;
+			background-color: var(--button--color-background) !important;
+			border-color: transparent;
+		}
+
+		&:focus {
```

Comment:
I think that removing this  focus style solves https://github.com/WordPress/twentytwentyone/pull/886#issuecomment-736542156 ?

Reply:
Should be working better with 27f1fe1 + dbf60c7:

![Kapture 2020-12-01 at 09 52 19](https://user-images.githubusercontent.com/1202812/100756184-f9a1db80-33ba-11eb-90d4-b137364a3ffc.gif)